### ISSUE TITLE
Support `null` type?

### ISSUE THREAD

**Issue by bokuweb** [dannynelson/json-schema-to-flow-type#5]
```
{
  "id": "json-rpc-sample",
  "$schema": "http://json-schema.org/draft-04/schema#",
  "definitions": {
    "foo": {
      "type": ["null", "string"]
    }
  }
}
```

Should this convert to `export type foo = string | null` or  `export type foo = ?string` ?
Now, error thrown.

```
    throw new Error("Invalid typeof value");
    ^

Error: Invalid typeof value
```

**Comment by dannynelson** [comment]
Hi @bokuweb, thanks for reporting this. The issue is fixed in `json-schema-to-flow-type@0.2.5`. Your schema should convert to:
```js
export type Foo = null | string;
export type JsonRpcSample = any;
```

**Comment by bokuweb** [comment]
@dannynelson thanks!! 🎉
